features/networking/sriov.feature

```diff
@@ -69,3 +71,347 @@ Feature: Sriov related scenarios
     Then the output should contain "192.168.2.206"
     And the output should contain "2001::2/64"
 
+  # @author zzhao@redhat.com
+  # @case_id OCP-21364
+  @destructive
+  @admin  
+  Scenario: Create pod with sriov-cni plugin and macvlan on the same interface
+    Given the sriov operator is running well
+    Given I obtain test data file "networking/sriov/sriovnetworkpolicy/intel-netdevice.yaml"
+    Given I create sriov resource with following:
+       | cr_yaml       | intel-netdevice.yaml     |
+       | cr_name       | intel-netdevice          |
+       | resource_type | sriovnetworknodepolicies |
+    Then the step should succeed
+    And I wait up to 300 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource    | sriovnetworknodestates           |
+      | namespace   | openshift-sriov-network-operator |
+      | o           | yaml                             |
+    Then the step should succeed
+    And the output should contain "intel-netdevice"
+    And the output should contain "syncStatus: Succeeded"
+    And the output should contain "vfID: 4"
+    """
+    Given I switch to the first user
+    And I have a project
+    And evaluation of `project.name` is stored in the :usr_project clipboard
+    Given I obtain test data file "networking/sriov/sriovnetwork/intelnetdevice.yaml"
+    Given I create sriov resource with following:
+       | cr_yaml       | intelnetdevice.yaml   |
+       | cr_name       | intel-netdevice-rhcos |
+       | resource_type | sriovnetwork          |
+       | project       | <%= cb.usr_project%>  |
+    Then the step should succeed
+
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource  | net-attach-def        |
+      | namespace | <%= cb.usr_project%>  |
+    Then the step should succeed
+    And the output should contain "intel-netdevice-rhcos"
+    """
+    Given I switch to the first user
+    And I use the "<%= cb.usr_project%>" project
```

Comment:
@zhaozhanqi Not sure you still need switch to the first user, no other users used after step #97

Reply:
@weliang1 Thank you for helping review this. you are right, no need to this step. also updated others scenario.